debian/copyright

```diff
@@ -0,0 +1,18 @@
+Files: debian/*
+Copyright: 2020 Nikesh Bernardshaw <nbernardshaw@clearpath.ai>
```

Comment:
Please also preserve whatever copyright the original author had on this work.

Reply:
Added the original author's github username and profile URL.